src/html/usage.rs

```diff
@@ -111,11 +113,32 @@ pub fn usage_to_md(
 
       usage_statement
     } else {
-      // when the imported symbol is a namespace import, we try to guess at an
-      // intelligent camelized name for the import based on the package name.
-      let import_symbol = "mod";
+      let module_import_symbol = ctx
+        .get_current_resolve()
+        .get_file()
+        .and_then(|file| {
+          ctx
+            .ctx
+            .doc_nodes
+            .get(file)
+            .and_then(|nodes| {
+              nodes
+                .iter()
+                .find(|node| node.kind() == DocNodeKind::ModuleDoc)
+            })
+            .and_then(|node| {
+              node.js_doc.tags.iter().find_map(|tag| {
+                if let JsDocTag::Module { name } = tag {
+                  name.as_ref().map(|name| name.to_string())
```

Comment:
I think the name might be causing a syntax error in import statement like `import * as foo-bar from "module";`. Can we avoid it somehow? (Maybe by removing non-alnum character from the name?)

Reply:
yea was thinking about that; I guess we could allow `[a-zA-Z$_]`, and replace any unknown characters with `_`. more is technically allowed, but i think this is good enough